stream_alert/rule_processor/sink.py

```diff
@@ -13,121 +13,89 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 '''
-
 import json
 import logging
 import sys
 
 import boto3
 from botocore.exceptions import ClientError
 
-_SNS_MAX_SIZE = (256*1024)
```

Comment:
There is still a [maximum Lambda invocation size](http://docs.aws.amazon.com/lambda/latest/dg/limits.html#limits-list) (6MB for asynchronous requests), but I don't think we'll be anywhere close to that limit. Just something to keep in mind